packages/api/lib/controllers/boxesController.js

```diff
@@ -487,6 +487,26 @@ const deleteBox = async function deleteBox (req, res, next) {
   }
 };
 
+const getAllTags = async function getAllTags (req, res, next) {
+  try {
+    const output = [];
+    const boxes = await Box.find().exec();
```

Comment:
Pretty sure there is a "cleaner" way of doing this , by only returning grouptags in the first place, but for now I have not been able to find this. Will use this for testing purposes on OSMX development, but I am happy for any input concering this

Reply:
You can use the `distinct` functionality of the `boxes` collection.
```javascript
const grouptags = await Box.find().distinct('grouptag').exec();
or
const grouptags2 = await Box.collection.distinct('grouptag');
```

Reply:
You can use the `distinct` functionality of the `boxes` collection.
```javascript
const grouptags = await Box.find().distinct('grouptag').exec();
or
const grouptags2 = await Box.collection.distinct('grouptag');
```